pandas/util/_validators.py

```diff
@@ -4,7 +4,7 @@
 """
 import warnings
 
-from pandas.core.dtypes.common import is_bool
+from pandas._libs.lib import is_bool
```

Comment:
this is the correct import; we really really don't want to import directly from .lib for things like this, these should *always* be from pandas.core.dtypes.common.

Reply:
this isn't a hill i want to die on, but the dependency structure consideration is that util._* are supposed to be "upstream" of core

Reply:
we don't enforce that in any way and this is a complete style break.

Reply:
ok, will change